README.md

```diff
@@ -121,6 +121,7 @@ BRC | Standard
 72   | [Protecting BRC-69 Key Linkage Information in Transit](./key-derivation/0072.md)
 73   | [Group Permissions for App Access](./wallet/0073.md)
 74   | [BSV Unified Merkle Path (BUMP) Format](./transactions/0074.md)
+75   | [Graph Aware Sync Protocol](./0075.md)
```

Comment:
SHould prefix the URL with ./transactions - and please update the BRC number to 76

Reply:
Please also add links to the transactions/README.md and SUMMARY.md files